neural_network_lyapunov/relu_to_optimization.py

```diff
@@ -611,6 +663,20 @@ def _compute_network_output_bounds(self,
                             network_input_up.detach().numpy(),
                             create_prog_callback,
                             binary_var_type)
+                if method == mip_utils.PropagateBoundsMethod.IA_MIP:
+                    if torch.abs(linear_output_lo[i] -
+                                 linear_output_lo_ia[i]) < 1E-3:
```

Comment:
why are we considering constraints active with such a higher threshold on the output layer?